node/core/runtime-api/src/cache.rs

```diff
@@ -41,6 +42,7 @@ const SESSION_INFO_CACHE_SIZE: usize = 64 * 1024;
 const DMQ_CONTENTS_CACHE_SIZE: usize = 64 * 1024;
 const INBOUND_HRMP_CHANNELS_CACHE_SIZE: usize = 64 * 1024;
 const CURRENT_BABE_EPOCH_CACHE_SIZE: usize = 64 * 1024;
+const ON_CHAIN_VOTES_CACHE_SIZE: usize = 3 * 1024;
```

Comment:
any reason for choosing 3KiB for cache size? the lowest we have is 64KiB, but it's not really backed by data

Reply:
No particular reason. We simply won't use it that much, so this could be an optimization to remove it.